gizmo.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import pandas as pd
+
+def hello(name, country='Finland'):
+    # These are two examples of how it can be done
+    # print("Hello {}, how are things in {}?".format(name, country))
+    print("Hello %s, how are things in %s?" % (name, country))
```

Comment:
or even `print(f"Hello {name}, how are things in {country}?")`

(this uses Python f-strings)